addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/workGenerators/WorkerGeneratorURLFuzz.java

```diff
@@ -188,18 +188,17 @@ public void run() {
                 Thread.sleep(3);
             }
         } catch (InterruptedException ex) {
-            Logger.getLogger(WorkerGeneratorURLFuzz.class.getName()).log(Level.SEVERE, null, ex);
+            LOG.error(ex.toString());
         } catch (MalformedURLException ex) {
-            Logger.getLogger(WorkerGeneratorURLFuzz.class.getName()).log(Level.SEVERE, null, ex);
+            LOG.error(ex.toString());
```

Comment:
I'd suggest warn, it could also include the exception.

Reply:
You mean `ex.getMessage`? Or `ex.toString(), ex`

Reply:
The latter, although the message could be hardcoded, e.g. "Failed to create the fuzzed URL:".

Reply:
Which one would be preferred? `ex.toString(), ex`, or hardcoded?

Reply:
When including the exception it includes the message already so better adding the hardcoded for more context.